[#993] Fix graph containsment issues in FlinkAsciiGraphLoader.getLogicalGraph

p-f opened the pull request:
* Set graph containment per default (use `getLogicalGraph(true)` instead of `getLogicalGraph(false)`)
* Use the `LogicalGraphFactory` to avoid duplicate code
* Set the correct graph head label after that

I'm not sure if we really need a default graph head label for the database graph. If not, then the `TransformationFunction` could be removed.

merando commented on gradoop-flink/src/main/java/org/gradoop/flink/util/FlinkAsciiGraphLoader.java:
```diff
@@ -136,17 +136,15 @@ public LogicalGraph getLogicalGraph() {
-        env.fromCollection(getEdges()).map(new AddToGraph<>(graphHead)));
-
+      return config.getLogicalGraphFactory().fromCollections(getVertices(), getEdges())
+        .transformGraphHead((head, transformed) -> {
```
I recently had some problems with lambdas during remote execution or shell based execution. Did you tested this one outside your IDE?

ChrizZz110 replied:
Optional approach:
```
private GraphHead setDefaultLabel(GraphHead current, GraphHead transformed) {
    current.setLabel(GradoopConstants.DB_GRAPH_LABEL);
    return current;
  }
```
An call it:
```
.transformGraphHead(this::setDefaultLabel);
```

p-f commented:
The return type of the lambda is explicitly set in `org.gradoop.flink.model.impl.operators.transformation.Transformation`, so that shouldn't be a problem.
I replaced the lambda with the predefined `RenameLabel` transformation and added a simple tests.
The `getLogicalGraph()` function was previously not directly tested (although it is used in multiple tests).
